Mathlib/AlgebraicTopology/SimplicialSet/Degenerate.lean

```diff
@@ -0,0 +1,123 @@
+/-
+Copyright (c) 2025 Joël Riou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joël Riou
+-/
+import Mathlib.AlgebraicTopology.SimplicialSet.Basic
+
+/-!
+# Degenerate simplices
+
+Given a simplicial set `X` and `n : ℕ`, we define the sets `X.degenerate n`
+and `X.nonDegenerate n` of degenerate or non-degenerate simplices of dimension `n`.
+
+## TODO (@joelriou)
+
+* `SSet.exists_nonDegenerate` shows that any `n`-simplex can be written
+as `X.map f.op y` for some epimorphism `f : [n] ⟶ [m]` and some
+non-degenerate simplex `y`. Show that `f` and `y` are unique.
+
+-/
+
+universe u
+
+open CategoryTheory Simplicial Limits Opposite
+
+namespace SSet
+
+variable (X : SSet.{u})
+
+/-- A `n`-simplex of a simplicial `X` is degenerate if it is in the range
```

Comment:
```suggestion
/-- An `n`-simplex of a simplicial set `X` is degenerate if it is in the range
```
I'm not actually certain if "an `n`-simplex" or "a `n`-simplex" is correct, but some primitive grepping suggests that the string "an \`n\`-" is far more common in mathlib than the string "a \`n\`-".